inlong-common/src/main/java/org/apache/inlong/common/enums/TaskTypeEnum.java

```diff
@@ -31,8 +31,10 @@ public enum TaskTypeEnum {
     MONGODB(9),
     TUBEMQ(10),
     REDIS(11),
-
-
+    CKAFKA(12),
+    MYSQL(13),
```

Comment:
Excuse me, what does the difference between the `SQL`, `BINLOG`, and `MYSQL`?

Reply:
delete MYSQL in line 35